monai/apps/auto3dseg/utils.py

```diff
@@ -63,6 +72,6 @@ def export_bundle_algo_history(history: list[dict[str, BundleAlgo]]) -> None:
     Args:
         history: a List of Bundle. Typically, the history can be obtained from BundleGen get_history method
     """
-    for task in history:
-        for _, algo in task.items():
-            algo_to_pickle(algo, template_path=algo.template_path)
+    for algo_dict in history:
+        algo = algo_dict[AlgoEnsembleKeys.ALGO]
```

Comment:
I guess after this PR, the user will see this enum "AlgoEnsembleKeys" even if they don't do ensemble, because that's the way to iterate over the Auto3DSeg-templated algorithms. 
Originally the keys are for ensemble but the scope will be expanded. Maybe we can consider another name, such as "Auto3DSegAlgoKeys" and deprecate `AlgoEnsembleKeys` to avoid confusion.
The deprecation doesn't have to be in this PR, and I can submit a PR for that after this one is merged.

Reply:
sure, may be something even shorter e.g. renaming AlgoEnsembleKeys -> AlgoKeys (after this PR)